src/posix/fs.cpp

```diff
@@ -1842,6 +1842,14 @@ DirNotify* PosixFileSystemAccess::newdirnotify(LocalPath& localpath, LocalPath&
     return dirnotify;
 }
 
+bool PosixFileSystemAccess::issyncsupported(const LocalPath& localpathArg, bool& isnetwork, SyncError& syncError, SyncWarning& syncWarning)
```

Comment:
Prior to this PR, the function took fewer parameters, and the base class had a default implementation in the header of just { return true; }, which is what Posix builds were using.   With the added parameters, I've defaulted that to =0 in the base class, and provided an implementation for posix (essentially still "return true" but sets the error parameters also).     The issyncsupported() function has been around since 2015 so I don't think we necessarily need to adjust per filesystem for Posix, unless we hear of some that are definitely not working?  Or perhaps if a ticket is created, it should be to investigate which filesystems do and don't work for Mac/Linux.  thanks

Reply:
Given the expected workload for the coming months, let's simply keep it in mind when a user complains about sync issues on Mac/Linux. A network/vbox drive and/or a FAT/HGFS formatted one could be the reason.